src/_includes/conf-social-share.html

```diff
@@ -31,4 +39,12 @@
       </svg>
     </a>
   </li>
+  <li>
+    <a class="hover:text-social-linkedin" href="{{ site.social.linkedin }}">
+      <svg class="w-8 h-8 icon">
+        <title>Facebook</title>
+        <use xlink:href="#linkedin-icon"></use>
+      </svg>
+    </a>
+  </li>
```

Comment:
Seems like you're adding a Facebook, but, the `href` value is pointing to `linkedin` and the icon in reference too seems to be`linkedin`

Reply:
Thanks for catching. I meant to remove Facebook.